### PR TITLE
Product detail page

### PR DESCRIPTION
**Pull request by rahulyadav139**
I have created the product details page. Kindly review it.

### CODE REVIEW

**Review by j24m — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -46,9 +46,15 @@
             </div>
           </li>
           <li>
-            <div class="avatar small">
+            <div class=" profile avatar small">
```
Comment (tarunsankhla) [inline index.html:49]: small ? text avatar image add a meaning full classname

File: product-detail.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  box-shadow: none;
+}
+.product-detail .image {
+  /* padding: 0 5rem; */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.product-detail .image img {
```
Comment (tarunsankhla) [inline product-detail.css:14]: why do you require both classname and tag, use anyone

File: products.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  /* font-size: 12px; */
+}
+.input-field-icon input {
+  background: transparent;
+}
+/* temp */
+
+.car-item ul {
+  display: none;
+  list-style: none;
```
Comment (tarunsankhla) [inline products.css:71]: if display is none then why list-style none ? I am not sure I think you wont require that and also card*

**Review by tarunsankhla — APPROVED** [review]

File: products.css (hunk 3 above)
Comment (rahulyadav139) [reply products.css:71]: Yes, the display is none by default but when users want to see this list they can see that I have made a radio input to do that.

File: product-detail.css (hunk 2 above)
Comment (rahulyadav139) [reply product-detail.css:14]: I used this just to link all parent classes.  If I want to give "image" class to any element then I can do that and it won't overlap with this.

File: index.html (hunk 1 above)
Comment (rahulyadav139) [reply index.html:49]: I want to keep additional (add-on) classes simple but I attached these classes to parent classes so these overlap with another.

**Comment by rahulyadav139** [comment]
Thank you for the reviews.

**Pull request merged**